seatunnel-connectors/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisOutputFormat.java

```diff
@@ -44,6 +44,7 @@
  **/
 public class DorisOutputFormat<T> extends RichOutputFormat<T> {
     private static final Logger LOGGER = LoggerFactory.getLogger(DorisSinkFunction.class);
+    private static final long DEFAULT_INTERVAL_MS = TimeUnit.MINUTES.toMillis(1);
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Why expand this value?

Reply:
You are right, it should be `TimeUnit.SECONDS.toMillis(1)`, let me fix it

Reply:
Done